.oscal/ssp.oscal.yaml

```diff
@@ -0,0 +1,162 @@
+system-security-plan:
+  uuid: 8c1ee2a6-bd14-4f77-b2f7-3d2bc0aaa55e
+  metadata:
+    title: "Oscal Server Software Development System Security Plan"
+    last-modified: "2024-10-24T00:00:00Z"
+    version: "1.0"
+    oscal-version: "1.0.4"
+    parties:
+      - uuid: 11111111-0000-4000-9000-100000000001
+        type: organization
+        name: "Metaschema Framework Orginization"
+        short-name: "MFO"
+    roles:
+      - id: system-owner
+        title: System Owner
+      - id: security-officer 
+        title: Security Officer
+    responsible-parties:
+      - role-id: system-owner
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000001
+      - role-id: security-officer
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000001
```

Comment:
Re a previous conversation, we really need to meet to discuss what these roles and what they would mean. To sit, SSDF doesn't really need or explicate those, that's a RMF SP 800-53 concept.

Let's put a meeting on the books to discuss this.